ext/linux/osq_lock.h

```diff
@@ -39,34 +42,67 @@ struct optimistic_spin_queue {
 /* 0 means thread unlocked, 1~N represents each individual thread on core 1~N */
 #define OSQ_UNLOCKED_VAL (0)
 
+/*
+ * maximum backoff sleep time in microseconds (default 10us)
+ * linux kernel scheduling intrinsic delay is less than 10us
+ * therefore osq may be rescheduled after this backoff time at best
+ * http://www.brendangregg.com/blog/2017-03-16/perf-sched.html
+ */
+#define MAX_SLEEP_US 10
+
+/*
+ * Maximum default unqueue_retry, most system spins at least 500~1000 times
+ * to unqueue.
+ */
+#define MAX_UNQUEUE_RETRY 1000000
```

Comment:
Is this actually a maximum or just the default?

Reply:
Just the default, I can change the name

Reply:
Please change the name to DEFAULT.